### ISSUE TITLE
Adapting the refresh rate makes video start a few seconds into the video

### ISSUE THREAD

**Issue by ghost** [yuliskov/SmartYouTubeTV#398]
I don't know if my tv doesn't switch the refresh rate quickly enough, but when adapting the video to the refresh rate by the time my tv switches to the rate the video already has started playing for a few seconds.

**Comment by yuliskov** [comment]
Is AFR enabled inside the app?

**Comment by ghost** [comment]
Yes Autoframerate is activated in the video setting. By the times it adjusts the video is already about 4 seconds in.

Edit: I think I wasn't using the last release, just reinstalled the app and it seems to be working fine now, I also noticed the option to pause while the refresh rate adjusts but I don't seem to be needing it.

Thanks for your help!
